src/auth/index.js

```diff
@@ -67,8 +65,11 @@ var AuthenticationClient = function(options) {
   };
 
   if (options.telemetry !== false) {
-    var telemetry = jsonToBase64(options.clientInfo || this.getClientInfo());
-    managerOptions.headers['Auth0-Client'] = telemetry;
+    var clientInfo = options.clientInfo || utils.generateClientInfo();
+    if ('string' === typeof clientInfo.name && clientInfo.name.length > 0) {
```

Comment:
if name is empty or not present, skip sending telemetry

Reply:
```suggestion
    if ('string' === typeof clientInfo.name && clientInfo.name) {
```

Reply:
```suggestion
    if ('string' === typeof clientInfo.name && clientInfo.name) {
```